frontend/src/routes/Home/Search/SearchPage.tsx

```diff
@@ -48,29 +49,39 @@ const useStyles = makeStyles({
     },
 })
 
-// Adds AcmAlert to page if there's errors from the Apollo queries.
 function HandleErrors(schemaError: ApolloError | undefined, completeError: ApolloError | undefined) {
     const { t } = useTranslation()
     const notEnabled = 'not enabled'
-    if (schemaError || completeError) {
+    if (schemaError?.message.includes(notEnabled) || completeError?.message.includes(notEnabled)) {
         return (
-            <div style={{ marginBottom: '1rem' }}>
-                <AcmAlert
-                    noClose
-                    variant={
-                        schemaError?.message.includes(notEnabled) || completeError?.message.includes(notEnabled)
-                            ? 'info'
-                            : 'danger'
-                    }
-                    isInline
-                    title={
-                        schemaError?.message.includes(notEnabled) || completeError?.message.includes(notEnabled)
-                            ? t('search.filter.info.title')
-                            : t('search.filter.errors.title')
-                    }
-                    subtitle={schemaError?.message || completeError?.message}
-                />
-            </div>
+            <EmptyState>
+                <EmptyStateIcon icon={InfoCircleIcon} color={'var(--pf-global--info-color--100)'} />
+                <Title size="lg" headingLevel="h4">
+                    {t('search.filter.info.title')}
+                </Title>
+                <EmptyStateBody>{schemaError?.message || completeError?.message}</EmptyStateBody>
```

Comment:
Do we always have one of these 2 error messages? I wonder if we should add `... || 'Default empty state message'`

Reply:
We will always have one or the other here - the conditional above checks that at least one is defined.